images/peer/Dockerfile

```diff
@@ -29,12 +29,13 @@ RUN make peer GO_TAGS=${GO_TAGS}
 RUN make ccaasbuilder
 
 FROM peer-base
+ARG MARCH
 ENV FABRIC_CFG_PATH /etc/hyperledger/fabric
 VOLUME /etc/hyperledger/fabric
 VOLUME /var/hyperledger
 COPY --from=peer /go/src/github.com/hyperledger/fabric/build/bin /usr/local/bin
 COPY --from=peer /go/src/github.com/hyperledger/fabric/sampleconfig/msp ${FABRIC_CFG_PATH}/msp
 COPY --from=peer /go/src/github.com/hyperledger/fabric/sampleconfig/core.yaml ${FABRIC_CFG_PATH}/core.yaml
-COPY --from=peer /go/src/github.com/hyperledger/fabric/release/linux-amd64/bin/ccaas_builder/bin/ /opt/hyperledger/ccaas_builder/bin/
+COPY --from=peer /go/src/github.com/hyperledger/fabric/release/${MARCH}/bin/ccaas_builder/bin/ /opt/hyperledger/ccaas_builder/bin/
```

Comment:
I do not think that will work since line 29 is running inside the staged docker image, and $MARCH will point to host architecture rather than the correct one.

Reply:
Agree, It should be set to stage docker image specific os-arch.  i think, the value should be read from the previous stage where ccaasbuilder is compiled.